packages/bpk-component-modal/src/bpk-modal-dialog.scss

```diff
@@ -98,9 +98,11 @@
 
   &__dialog-heading {
     margin: 0;
-    font-weight: 700;
 
     @include bpk-text-base;
```

Comment:
If a consumer uses `@include bpk-text-bold;` in an ios webview then thats a mistake right? I.e. the designer shouldn't have given them that design?

Otherwise the value for `font-weight` in a `bpk-text-bold` mixin should be the `FONT_WEIGHT_BOLD` token?